libs/dh/admin/feature-user-roles/src/lib/drawer/tabs/dh-drawer-role-tabs.component.html

```diff
@@ -22,7 +22,7 @@
     <watt-tab [label]="t('permissions.tabLabel')">
       <dh-role-permissions [role]="role" />
     </watt-tab>
-    <watt-tab *dhPermissionRequired="['user-roles:manage']" [label]="t('history.tabLabel')">
+    <watt-tab *dhPermissionRequired="['fas']" [label]="t('history.tabLabel')">
```

Comment:
suggestion: I can remember that some "History" tabs lazy-load their data (first when the tab is open), while others load data when the drawer gets open. Can you test whether history queries are not executed when the drawer gets open. Also test what happens when a user/actor is updated. History queries are refetched after that as well.